Templates/WebHook-GitHub/index.js

```diff
@@ -1,4 +1,8 @@
 ﻿module.exports = function (body, context) {
     context.log('GitHub WebHook triggered! ' + body.comment.body);
-    context.done(null, 'New GitHub comment: ' + body.comment.body);
+    context.done(null, {
+        res: {
+            body: 'New GitHub comment: ' + JSON.stringify(body.comment.body, null, " ")
```

Comment:
That feels really ugly. But we know we need to improve this. I want `context.done(null, body.comment.body);`